app/assets/javascripts/import_records/ImportRecordsPage.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import GenericLoader from '../components/GenericLoader';
+import {apiFetchJson} from '../helpers/apiFetchJson';
+import ImportRecordCard from './ImportRecordCard';
+import QueuedJobCard from './QueuedJobCard';
+
+export default class ImportRecordsPage extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.fetchImportRecords = this.fetchImportRecords.bind(this);
+    this.renderPage = this.renderPage.bind(this);
+  }
+
+  fetchImportRecords() {
+    return apiFetchJson('/api/import_records');
+  }
+
+  render() {
+    return (
+      <div className="ImportRecordsPage">
+        <GenericLoader
+          promiseFn={this.fetchImportRecords}
+          render={this.renderPage} />
+      </div>
+    );
+  }
+  renderImportRecords(importRecords) {
+    const divStyle = {margin: '15px 0'};
```

Comment:
And in general, does it seem okay to factor out the static styles up to the top or bottom of the file, outside of the component itself (eg, [one possible example](https://github.com/studentinsights/studentinsights/blob/master/app/assets/javascripts/school_courses/SchoolCoursesPage.js#L141))?  It may not particularly matter for what we're doing here, but seems good to keep them somewhat separate (but still in the same file for easy editing and since they're tightly coupled).

Reply:
Yeah, that sounds like a good practice and allowing for re-use within the file is a good idea. Kind of quasi-CSS-like in the sense that we're separating styles from elements, albeit on the same page. 

I like putting the styles at the bottom of the file as opposed to the top, that way when you open the file for the first time you still see the component name and constructors first.